lightning/src/ln/channelmanager.rs

```diff
@@ -8584,12 +8584,14 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 		let entropy = &*$self.entropy_source;
 		let secp_ctx = &$self.secp_ctx;
 
-		let path = $self.create_blinded_path().map_err(|_| Bolt12SemanticError::MissingPaths)?;
+		let paths = $self.create_blinded_path(5)
```

Comment:
Hmmm... I don't think we want this. This was dramatically increase the size of the QR code size, making it difficult to scan. We should keep this to one. Later we can always add a parameter to `create_offer_builder` if we want to allow users to include more paths.

Reply:
Sounds reasonable!
Reverted the changes to only create one path for offer, and refund in [f32a93a](https://github.com/lightningdevkit/rust-lightning/commit/f32a93acc60d20078c42b96dbb4466969b06f5b3)
Also added a TODO comment so that we can keep track of this in the codebase!
Thanks!